src/ngAnimate/animation.js

```diff
@@ -186,8 +184,9 @@ var $$AnimationProvider = ['$animateProvider', /** @this */ function($animatePro
         forEach(groupedAnimations, function(animationEntry) {
           var element = animationEntry.from ? animationEntry.from.element : animationEntry.element;
           var extraClasses = options.addClass;
+
           extraClasses = (extraClasses ? (extraClasses + ' ') : '') + NG_ANIMATE_CLASSNAME;
-          var cacheKey = $$animateCache.cacheKey(node, event, extraClasses, options.removeClass);
+          var cacheKey = $$animateCache.cacheKey(element[0], animationEntry.event, extraClasses, options.removeClass);
```

Comment:
AFAICT, `element` could also be a DOM node (in which case `element[0]` will be undefined), no?

Reply:
As far as I can see, at this point it's already normalized to be a DOM node

Reply:
Yeah, looking at it again, it seems that `element` is treated as a jqLite/jQuery element. So, it's fine.